.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+on:
+  # Trigger analysis when pushing to your main branches, and when creating a pull request.
+  push:
+    branches:
+      - main
+      - master
+      - develop
+      - 'releases/**'
+  pull_request:
+      types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'yarn'
+      - run: yarn install --frozen-lockfile
+  lint:
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'yarn'
+      - run: yarn install --frozen-lockfile
+      - run: yarn lint
+  test:
+    runs-on: ubuntu-latest
+    needs: [lint]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'yarn'
+      - run: yarn install --frozen-lockfile
```

Comment:
Esto del `setup-node` y el `yarn install` se podría meter en una custom action. También se podría agregar cacheo de las dependencias. Creo que podrías usar directamente [esto](https://github.com/amalgamaco/react-native-template/blob/main/.github/actions/install-yarn-dependencies/action.yml) que hicimos para el template de react native.

Reply:
Las podes publicar en el [marketplace de github](https://docs.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace) o compartirlas [solo dentro de la organización](https://docs.github.com/en/actions/creating-actions/sharing-actions-and-workflows-with-your-organization). O si queres por ahora copiar y pegar la custom action adentro de `.github/actions`.

Reply:
La verdad intenté compartirlas dentro de la organización, pero no pude jaja. Podríamos en algún momento subir todas nuestras actions al market place, esta buena esa idea. Por ahora metí la custom action acá.